contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,162 @@
+
+### Affordable
+The free plan offers 2000 minutes of build time per month for public repositories and a pay-per-minute plan for further use.
+
+---
+
+## How does Github Actions work
+<!--This is a simple high-level overview of the tool to give the reader some concrete sense of X (as opposed to limiting to an entirely abstract description). It's useful to give concrete examples such as code examples. Do not try to 'teach' how to use the X (assuming your in the style of a tutorial. If the tool is worth learning, there must be good tutorials about it already.-->
+### Concepts
+#### Workflow
+A workflow describes a sequence of jobs to be run when a trigger event occurs. Each job consists of a series of steps, which may be actions or terminal commands. Each workflow is defined as a `.yml` file in the `.github/workflows` directory of a repository.
+
+#### Action
+An action is a piece of code that performs one single modular task. For example, [checking out a repository](https://github.com/actions/checkout).
+
+#### Triggers
+Workflows can be triggered by events on the Github repository, ranging from opening pull requests or issues, making commits to scheduled events, or even from an external event by calling Github's REST API endpoint.
+
+
+### How is Github Actions being used
+
+#### Simple CI/CD workflow example for a NodeJS app
+1. Trigger when a commit/pull request is made to master
+2. Build
+3. Run tests on ubuntu
+4. Deploy app if tests pass
+
+##### [Workflow file](https://github.com/actions/starter-workflows/blob/master/ci/node.js.yml)
+```yaml
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - run: echo Hello World!
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build --if-present
+    - run: npm test
+      env:
+        CI: true
+    - run: npm run deploy
+```
+
+
+
+## How to Get Started with X?
+<!--
+Provide a learning path for the reader. Try to give one good learning path rather than many random resources.
+-->
+The [official documentation](https://help.github.com/en/actions/getting-started-with-github-actions/overview) provides a useful overview of Github Actions (summarized in this article), but it does not give you a clear and easy to follow path to getting started. This is a mini-guide for you to get started with Github Actions.
+
+### Set up a simple workflow (Node.js)
+Fortunately for us, Github has made it extremely easy to set up a simple workflow without even leaving their website. 
+
+![](https://i.imgur.com/zC4ScYn.png)
```

Comment:
Consider pushing the static asset to the repo instead of linking to an external service.